src/frontend/src/lib/utils/balances.utils.ts

```diff
@@ -1,10 +1,26 @@
 import { type BalancesData } from '$lib/stores/balances.store';
 import type { CertifiedStoreData } from '$lib/stores/certified.store';
 import type { TokenId } from '$lib/types/token';
+import type { Option } from '$lib/types/utils';
 import { nonNullish } from '@dfinity/utils';
 
 export const checkAnyNonZeroBalance = ($balancesStore: CertifiedStoreData<BalancesData>): boolean =>
 	nonNullish($balancesStore) &&
 	Object.getOwnPropertySymbols($balancesStore).some(
 		(tokenId) => !($balancesStore[tokenId as TokenId]?.data?.isZero() ?? true)
 	);
+
+export const checkAllBalancesZero = ({
+	$balancesStore,
+	minLength
+}: {
+	$balancesStore: CertifiedStoreData<BalancesData>;
+	minLength: number;
+}): boolean =>
+	nonNullish($balancesStore) &&
+	Object.getOwnPropertySymbols($balancesStore).length >= Math.max(minLength, 1) &&
```

Comment:
I was wondering if using directly `Object.values` could be more effective or simplify the code?

Reply:
`Object.values` does not work for `symbols` in the keys unfortunately... only `string`

same for `Object.keys`

> The Object.values() static method returns an array of a given object's own enumerable string-keyed property values.

from [documentaiton](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values).